api-reference/10 UI Widgets/dxScheduler/1 Configuration/currentView.md

```diff
@@ -15,6 +15,8 @@ When more than one view matches the **currentView** value, the **Scheduler** dis
 
 To subscribe to changes of the current view, use the [onOptionChanged](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onOptionChanged) handler.
 
+You can to scroll to a specified position in the current view. For this, use the [scrollTo(date, group, allDay)](/Documentation/ApiReference/UI_Widgets/dxScheduler/Methods/#scrollTodate_group_allDay) method.
+
```

Comment:
scrollTo has a very narrow pool of use-cases, there's no reason to mention it in this article. You can add a link to the See Also section in the scrolling description: https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/scrolling/